tests/bluetooth/tester/src/bttester.h

```diff
@@ -228,6 +229,17 @@ struct gap_passkey_confirm_cmd {
 	u8_t match;
 } __packed;
 
+#define GAP_START_DIRECTED_ADV		0x15
+struct gap_start_directed_adv_cmd {
+	u8_t address_type;
+	u8_t address[6];
```

Comment:
In hci.h we would usually use `bt_addr_le_t` for this. Should we do that here as well?
It allows us to bt_addr_le_copy for example.

Reply:
If we do, then let's do it for all of the commands and events that use this, and in a later commit that cleans this up. This is just a quick update to the latest spec.

Reply:
Fair enough.